net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -28,8 +28,8 @@ DEPEND="${RDEPEND}
 	i2p-hardening? ( >=sys-devel/gcc-4.7 )
 	|| ( >=sys-devel/gcc-4.7 >=sys-devel/clang-3.3 )"
 
-I2PD_USER="${I2PD_USER:-i2pd}"
-I2PD_GROUP="${I2PD_GROUP:-i2pd}"
+I2PD_USER=i2pd
+I2PD_GROUP=i2pd
 
```

Comment:
In our systemd service file we have
```
User=i2pd
Group=i2pd
```
This is static. Is there a way to use environment variables there?